src/VisualStudio/LiveShare/Impl/Client/CodeActions/RoslynRemoteCodeAction.cs

```diff
@@ -105,7 +105,7 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
             foreach (var changePair in _codeActionWorkspaceEdit.Changes)
             {
                 var documentName = await _lspClient.ProtocolConverter.FromProtocolUriAsync(new Uri(changePair.Key), true, cancellationToken).ConfigureAwait(false);
-                var doc = newSolution.GetDocumentFromURI(documentName);
+                var doc = newSolution.GetDocuments(documentName).First();
```

Comment:
good to add a comment here noting that since this is the client side, it doesn't have the right contexts anyway and is just applying edits to the physical file.  So taking the first is fine.